_posts/2024-01-10-practical_zephyr_basics.md

```diff
@@ -429,18 +429,18 @@ manifest.file=west.yml
 zephyr.base=zephyr
 build.board=nrf52840dk_nrf52840
 # It is also possible to delete an option
-# e.g., using `west config -d build.board`
+#, e.g., using `west config -d build. board`
```

Comment:
This should really be `build.board`.

Reply:
Ooops not sure what happened here. Maybe an autoformat.